### ISSUE TITLE
[Feature request] Share builds without saving them to your accout first

### ISSUE THREAD

**Issue by gfx687** [drg-tools/karl.gg#377]
First of all, thank you for a great site, it is far simpler to check some of the information about the game via opening karl.gg than wiki

**TL;DR**: share builds via link without saving them first to save the effort of logging in / saving builds / deleting them later

**Why**: to share one-time builds without writing them as numbers or clogging your saved builds list with throw-away stuff

This feature exists in Vermintide 2 build website and is very popular / widely used, you can see people share links to builds like this all the time, which doesn't really happen with karl.gg in DRG discords, at least the ones that I use. I see karl.gg mentioned for linking builds once in a blue moon (it is often used for checking weapon mods / OCs / etc however, which is great and significantly faster than visiting wiki for the same info)

In general, it will lower the effort required and will make it more likely that people will use the website for sharing builds

**Example of similar functionality** -
https://www.ranalds.gift/heroes/1/112321/33,3,4,6/12,4,5,2/3,2,1/5,4,4/7,2,3

above is a link to a complete build that I did not save to my account, all the info about the build is stored inside the URL.

For example the first "1" we see in the link is the career, six numbers after are talents, etcetera

**Comment by Darthmaul** [comment]
Hi @gfx687  thanks for the suggestion. To clarify - this is requesting the ability to share a set build without actually saving it, right?

This is a good suggestion and we've discussed it before. It would take some doing, but definitely worth keeping in mind. We can take a look at it for future implementation 👍
